src/explore-education-statistics-common/src/components/CollapsibleList.tsx

```diff
@@ -1,78 +1,72 @@
 import ButtonText from '@common/components/ButtonText';
 import useToggle from '@common/hooks/useToggle';
 import classNames from 'classnames';
-import React, {
-  Children,
-  cloneElement,
-  isValidElement,
-  ReactNode,
-} from 'react';
-import styles from './CollapsibleList.module.scss';
+import React, { Children, createElement, ReactNode } from 'react';
 
 interface Props {
   children: ReactNode;
   collapseAfter?: number;
+  id: string;
+  itemName?: string;
+  itemNamePlural?: string;
   isCollapsed?: boolean;
   listStyle?: 'none' | 'number' | 'bullet';
-  testId?: string;
 }
 
 const CollapsibleList = ({
   children,
   collapseAfter = 5,
+  id,
+  itemName = 'item',
+  itemNamePlural: initialItemNamePlural,
   isCollapsed = true,
   listStyle = 'none',
-  testId,
 }: Props) => {
   const [collapsed, toggleCollapsed] = useToggle(isCollapsed);
 
-  const listClasses = classNames('govuk-list', styles.listContainer, {
+  const itemNamePlural = initialItemNamePlural ?? `${itemName}s`;
```

Comment:
I thought about making itemNamePlural required if you specify an item name but as in all existing cases where it's used adding 's' does work it seemed a bit unnecessary, so went with making itemNamePlural optional so you can override the default pluralisation if needed.

Reply:
My only concern would be that some future person would potentially neglect to set the `itemNamePlural` based on the existing examples not setting it. Of course, testing should flag this up, but this could be easily missed if the collapse threshold isn't hit.

Personally, I think the minor inconvenience of having a type that enforces this is worth not having this come back as a bug in the future.

Reply:
Fair enough, I've changed it now.